metrictank-sample.ini

```diff
@@ -269,6 +269,9 @@ mode = single
 min-available-shards = 0
 # How long to wait before aborting http requests to cluster peers and returning a http 503 service unavailable
 http-timeout = 60s
+# GOGC value to use when node is not ready.  Defaults to GOGC
+# you can use this to set a more aggressive, latency-inducing GC behavior when the node is initializing and hungry for extra memory
+# gc-percent-not-ready = 100
```

Comment:
Is this intentionally commented? I guess it makes no difference, as MT will just use its internal default anyway, I was just wondering because most other settings in the file are not commented.

Reply:
it is intentional. the difference with all other settings is that for this one, the default value is dynamically set based on GOGC. (see ConfigSetup in cluster/config.go)